services/npm/npm-unpacked-size.service.js

```diff
@@ -0,0 +1,83 @@
+import Joi from 'joi'
+import prettyBytes from 'pretty-bytes'
+import { pathParam, queryParam } from '../index.js'
+import { optionalNonNegativeInteger } from '../validators.js'
+import NpmBase, { packageNameDescription } from './npm-base.js'
+
+const schema = Joi.object({
+  dist: Joi.object({
+    unpackedSize: optionalNonNegativeInteger,
+  }).required(),
+}).required()
+
+export default class NpmUnpackedSize extends NpmBase {
+  static category = 'size'
+
+  static route = this.buildRoute('npm/unpacked-size', { withTag: true })
+
+  static openApi = {
+    '/npm/unpacked-size/{packageName}': {
+      get: {
+        summary: 'NPM Unpacked Size',
+        parameters: [
+          pathParam({
+            name: 'packageName',
+            example: 'npm',
+            description: packageNameDescription,
+          }),
+          queryParam({
+            name: 'registry_uri',
+            example: 'https://registry.npmjs.com',
+          }),
+        ],
+      },
+    },
+    '/npm/unpacked-size/{packageName}/{version}': {
+      get: {
+        summary: 'NPM Unpacked Size (with version)',
+        parameters: [
+          pathParam({
+            name: 'packageName',
+            example: 'npm',
+            description: packageNameDescription,
+          }),
+          pathParam({
+            name: 'version',
+            example: '4.18.2',
+          }),
+          queryParam({
+            name: 'registry_uri',
```

Comment:
Also added a `registry_uri` query param as they were existent on the other NPM-based services.